### PR TITLE
transition-added-to-sections

### PR DESCRIPTION
**Pull request by sricharanandra**
Issue #30  
This issue was raised requesting transition effects between the sections in the homepage to enhance the visual experience.

This adds a transition effect which make jumping between sections via the navbar smoother, by adding a scroll behaviour to the entire page in the style.css file.

### CODE REVIEW

**Comment by hrushikeshrv** [comment]
Hi @sricharanandra, thanks for the PR. Although I'm not sure if this PR will count under Hacktoberfest since it seems too trivial. 

I've left a review comment. Please check 👍🏻

File: docs/styles.css [hunk 1]
```diff
@@ -245,4 +245,8 @@ div.code-block {
 
 #api h1, #api h2, #api h3, #api h4, #api h5, #api h6 {
     font-family: var(--monospace-font);
+}
+
+html {
+    scroll-behavior: smooth;
```
Comment (hrushikeshrv) [inline docs/styles.css:251]: There is an `html` selector at line 87 of this file, please add the scroll behavior there.

**Review by hrushikeshrv — CHANGES_REQUESTED** [review]

File: docs/styles.css (hunk 1 above)
Comment (sricharanandra) [reply docs/styles.css]: I've made the suggested changes in a new commit.

**Comment by hrushikeshrv** [comment]
Thanks @sricharanandra 👍🏻

**Pull request merged**